src/Kdyby/Translation/CatalogueCompiler.php

```diff
@@ -60,10 +67,22 @@ public function enableDebugMode()
 	}
 
 
+	/**
+	 * Replaces cache storage with simple memory storage (per-request).
+	 */
+	public function disableDebugMode()
+	{
+		$this->cache = new Cache($this->cacheStorage, 'Kdyby\\Translation\\Translator');
+	}
+
 
-	public function invalidateCache()
+	public function invalidateCache($onlyTranslation = FALSE)
```

Comment:
proč bys to vůbec nedělal? Nechtěl jsem ti rozbít stávající chování (proto nepovinný parametr), a nevím, kde tam invaliduješ celou cache. Jestli nikdy nechceš mazat všechno, tak co tam ta metoda dělá?
A proč nemazat vždy podle tagu? Proč u FileCache sahat přímo na složku?